Replicator/tests/ReplicatorCollectionTest.cc

```diff
@@ -178,87 +178,167 @@ TEST_CASE_METHOD(ReplicatorCollectionTest, "Use Zero Collections", "[Push][Pull]
     _expectedError = {LiteCoreDomain, kC4ErrorInvalidParameter};
     runPushPullReplication({}, {});
 }
+#endif
+
+#define DISABLE_PUSH_AND_PULL
+#define DISABLE_CONTINUOUS
+
+static std::set<string> getDocInfos(C4Database* db, C4CollectionSpec coll) {
+    std::set<string> ret;
+    C4Collection* collection = ReplicatorCollectionTest::getCollection(db, coll);
+    auto e = c4coll_enumerateAllDocs(collection, nullptr, ERROR_INFO());
+    {
+        TransactionHelper t(db);
+        while (c4enum_next(e, ERROR_INFO())) {
+            C4DocumentInfo info;
+            c4enum_getDocumentInfo(e, &info);
+            alloc_slice docID(info.docID);
+            alloc_slice revID(info.revID);
+            string entry = docID.asString()+"/"+revID.asString();
+            ret.insert(entry);
+        }
+    }
+    c4enum_free(e);
+    return ret;
+}
+
+
+struct CheckDBEntries {
+    CheckDBEntries(C4Database* db, C4Database* db2, vector<C4CollectionSpec> specs)
+    : _db(db)
+    , _db2(db2)
+    {
+        for (auto i = 0; i < specs.size(); ++i) {
+            _collSpecs.push_back(specs[i]);
+            _dbBefore.push_back(getDocInfos(_db, specs[i]));
+            _db2Before.push_back(getDocInfos(_db2, specs[i]));
+        }
+    }
+    
+    ~CheckDBEntries() {
+        vector<set<string>> dbAfter;
+        vector<set<string>> db2After;
+        for (auto i = 0; i < _collSpecs.size(); ++i) {
+            dbAfter.push_back(getDocInfos(_db, _collSpecs[i]));
+            db2After.push_back(getDocInfos(_db2, _collSpecs[i]));
+            CHECK(dbAfter[i].size() == _dbBefore[i].size());
```

Comment:
Not sure about the purpose of checking the same size here. For the case of push-pull at least, the size will not be the same.

Reply:
A simple alternative to this class is a function to compare the end result at the end of the replication, whether same collection from the two dbs have the same docs or not, and check the number of docs.

Such as `compareCollections(roses, 20)` or `compareCollections({roses, tulips}, {10, 20})`

Reply:
I will come to that when working on Push and Pull. Right now, the counts do not match yet.